playbooks/compute/deploy-nto-gotest.yml

```diff
@@ -71,9 +71,10 @@
           export KUBECONFIG="{{ kubeconfig }}"
           export ROLE_WORKER_CNF=worker-cnf
           export PERF_TEST_PROFILE=performance
-          export IMAGE_REGISTRY="registry.redhat.io/"
+          export IMAGE_REGISTRY="registry.stage.redhat.io/"
```

Comment:
What's the reason for using the stage registry here?

Reply:
requested by the QE team